test/test_webdriver_utils.py

```diff
@@ -26,10 +26,11 @@ def test_parse_neterror_integration(self):
         manager.close()
         # I couldn't figure out how to just get a the get_command
         # Because the type has too many quotes
```

Comment:
@LordReigns Please remove this comment. Rest looks good.

Reply:
Oh k. I will remove it